Gruntfile.js

```diff
@@ -1,4 +1,14 @@
 "use strict";
+var fs = require('fs');
+var path = require('path');
+var http = require('http');
+var build = (function(){
+	// travis testing
+	if (process.env && process.env.BUILD) return process.env.BUILD == 'default' ? 'all' : 'nocompat'; 
+	// local testing
+	else return process.argv[2] == null || process.argv[2] == 'all' ? 'all' : 'nocompat';
+})();
+require('./Tests/httpServer.js')(build);
```

Comment:
@arian the idea is to serve HTML pages to the specs. Like these: https://github.com/SergioCrisostomo/mootools-core/tree/fix-2682/Tests/DOMReady

It could also be used in specs for Slick in the future. If there is a better way let me know. If this is a bad idea let me know also :smile: